js/memory-matching.js

```diff
@@ -1,19 +1,27 @@
-
-
 var matchingGame = {
-	elapsedTime: 0
+    elapsedTime: 0
+  , github: new GitHub()
 };
 
 matchingGame.deck = []
 
-// Like this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
-
-Hubbers["hubbers"].sort(shuffle).slice(0,8).map(function(hubber){
-	matchingGame.deck.push(hubber)
-	matchingGame.deck.push(hubber)
-});
+function getHubbers(callback) {
+   matchingGame.github.get("orgs/github/members", { all: true }, function (err, data) {
+        if (err) {
+            console.warn(err);
+            data = window.Hubbers;
+        }
+        callback(null, { hubbers: data });
+    });
+}
 
-console.log(matchingGame.deck)
+function shuffleHubbers(hubbers) {
+    // Like this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
+    hubbers.hubbers.sort(shuffle).slice(0,8).map(function(hubber){
```

Comment:
@alysonla This is not really random. :laughing: It's cute, but not really good solution to shuffle elements. See [this comment](http://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array#comment29711767_18650169):

> Downvoting as this isn't really that random. I don't know why it has so many upvotes. Do not use this method. It looks pretty, but isn't completely correct. Here are results after 10,000 iterations on how many times each number in your array hits index [0] (I can give the other results too): 1 = 29.19%, 2 = 29.53%, 3 = 20.06%, 4 = 11.91%, 5 = 5.99%, 6 = 3.32% 

This happens for an array like `[1, 2, 3, 4, 5, 6]`–so, in our case (~200 GitHub public members), the first ones (you are one of them, since they are sorted alphabetically probably :smile: ) will have the greatest chance to appear in the blocks, while the last ones will have the lowest chance to appear in the game. Black magic math! :trollface: